AutomatedTesting/Levels/ShaderReloadTest/billboard_visualize_rtt.material

```diff
@@ -0,0 +1,21 @@
+{
+    "materialType": "@gemroot:Atom_Feature_Common@/Assets/Materials/Types/StandardPBR.materialtype",
+    "materialTypeVersion": 5,
+    "propertyValues": {
+        "baseColor.color": [
+            0.800000011920929,
+            0.800000011920929,
+            0.800000011920929,
+            1.0
+        ],
+        //"baseColor.textureMap": "Levels/ShaderReloadTest/billboard_visualize_rtt.attimage",
```

Comment:
Did you mean to have '//' on this line?

Reply:
Yes. But forgot to add this comment:
```
// Commenting this out because it causes the `AssetProcessor` to never complete the compilation of this material.
// The solution is to run the Editor and open the `Material Instance Editor` and pick this same attachment image for the
// `baseColor.textureMap`. The actual GUID of the asset is actually stored in `ShaderLoadTest.prefab` (which is how // Material Instance Properties are stored). 
```